headers/types/ground_mode/version_dep_enums.h

```diff
@@ -0,0 +1,2476 @@
+// Version-dependent enums for ground mode
+
+#ifndef HEADERS_TYPES_GROUND_MODE_VERSION_DEP_ENUMS_H_
+#define HEADERS_TYPES_GROUND_MODE_VERSION_DEP_ENUMS_H_
+
+// North American version
```

Comment:
Idea: You could use a `define`d constant to check the region and set the enums just for the selected region.

Reply:
I thought about that, but decided against it since that would make these headers slightly harder to use. You'd either have to modify the region def every time manually before importing, pass it as a preprocessor arg, or I'd have to generate near-identical versions of the headers for each region. I'm hoping I don't run into any region-dependent struct layouts that require that sort of thing and can't be solved with unions 😅.